services/QuillLMS/engines/evidence/lib/tasks/benchmark.rake

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'benchmark'
+
+namespace :benchmark do
+
+  desc "Runs a benchmark on the Plagiarism algorithm"
+
+  task :plagiarism => :environment do |t, args|
```

Comment:
I like these reusable benchmarks! This will be very helpful if we need to optimize more.